pontoon/settings/base.py

```diff
@@ -782,6 +782,10 @@ def _allowed_hosts():
 # calendars according to the current locale
 USE_L10N = False
 
+#Enable Bugs tab on the team pages, pulling data from bugzilla.mozilla.org.
+#See bug 1567402 for details. A Mozilla-specific variable.
```

Comment:
Please add spaces after `#`'s. The lack of them make tests fail, as you can see by the [Travis report](https://travis-ci.org/mozilla/pontoon/builds/594658552?utm_source=github_status&utm_medium=notification).